src/Sylius/Bundle/AdminBundle/Resources/private/sass/_ui.scss

```diff
@@ -97,6 +96,51 @@ a {
         background: $sylius-brand-color !important;
         border-radius: 0 99px 99px 0 !important;
     }
+    .item {
+        padding-bottom: 5px;
+        .menu {
+            visibility: hidden;
+            overflow: hidden;
+            max-height: 0;
+            transition: all .5s ease-in-out;
```

Comment:
https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108
https://stackoverflow.com/a/12347702/6796253
These links seem to fix the actual problem. :)